src/frontend/src/flows/dappsExplorer/index.ts

```diff
@@ -47,12 +49,10 @@ const dappsExplorerTemplate = ({
 export const dappsExplorerPage = renderPage(dappsExplorerTemplate);
 
 /* Template for a single dapp */
-const dappTemplate = ({
-  link,
-  logo,
-  name,
-  oneLiner,
-}: DappDescription): TemplateResult => {
+const dappTemplate = (
+  { link, logo, name, oneLiner }: DappDescription,
+  copy: any
```

Comment:
@nmattia I might need help with that, I was unsure how to declare the correct type here.

Reply:
you can pass `{ open_cta: DynamicKey }` I think, might be the easiest